event-sourcing/event-sourcing-core/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+plugins {
+    id 'java-library'
+    id 'scala'
+}
+
+configurations {
+    tests
+}
+
+dependencies {
+    api project(':event-sourcing:event-sourcing-event-store-api')
+    api project(':event-sourcing:event-sourcing-pojo')
```

Comment:
this one is already provided by `event-sourcing:event-sourcing-event-store-api` as an api dependency I guess

Reply:
Yes. I've started 

> I think you use `implementation` deps too much and it avoids the good side of transitive dependencies (of course the wrong side would be to expose too much apis and create bad implicit dependencies)

Thanks for the review. 
This was a bit on purpose. 
There are too many modules IMO and I'm not familiar with their dependencies. 
We should improve this over time.
But we should be mindfull of what we consider API and what not.
In later modules I migrated I've put API all project modules, but not external libraries. 
I think this will help us make dependencies more visible - because we HAVE to deal and add each one. 
We can leave them like this and improve the situation when we re-organize the modules.